.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+          profile: minimal
+          override: true
+      - name: Build release binary
+        run: cargo build --release
+      - name: Package binary
+        run: |
+          mkdir dist
+          cp target/release/mdtablefix dist/mdtablefix-linux
+          cd dist
+          tar -czf mdtablefix-linux.tar.gz mdtablefix-linux
+      - name: Upload to release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)